src/machine/springSupport.ts

```diff
@@ -14,154 +14,177 @@
  * limitations under the License.
  */
 
-import { GitHubRepoRef } from "@atomist/automation-client/operations/common/GitHubRepoRef";
-import { GitProject } from "@atomist/automation-client/project/git/GitProject";
+import {GitHubRepoRef} from "@atomist/automation-client/operations/common/GitHubRepoRef";
 import {
-    allSatisfied,
-    ExecuteGoalResult,
-    ExecuteGoalWithLog, FromAtomist, goalContributors,
-    hasFile, not,
-    RunWithLogContext,
-    SoftwareDeliveryMachine, ToDefaultBranch, whenPushSatisfies,
+    Builder,
+    BuildGoal,
+    Deployer,
+    Goal,
+    goalContributors,
+    JustBuildGoal,
+    LocalDeploymentGoal,
+    not,
+    SoftwareDeliveryMachine,
+    whenPushSatisfies,
 } from "@atomist/sdm";
 import {
-    DefaultDockerImageNameCreator,
-    DockerBuildGoal,
-    DockerOptions,
-    executeDockerBuild,
-    executeVersioner, HasDockerfile, NoGoals,
-    tagRepo, ToPublicRepo,
+    DisableDeploy,
+    EnableDeploy,
+    executeVersioner,
+    LocalEndpointGoal,
+    LocalUndeploymentGoal,
+    lookFor200OnEndpointRootGet,
+    ManagedDeploymentTargeter,
+    ManagedDeploymentTargetInfo,
+    NoGoals,
+    StartupInfo,
+    tagRepo,
     VersionGoal,
 } from "@atomist/sdm-core";
 import {
     ProjectVersioner,
-    readSdmVersion,
 } from "@atomist/sdm-core/internal/delivery/build/local/projectVersioner";
 import {
     IsMaven,
     MavenProjectIdentifier,
     springBootGenerator,
     springBootTagger,
 } from "@atomist/sdm-pack-spring";
-import {CommonJavaGeneratorConfig, HasSpringBootApplicationClass} from "@atomist/sdm-pack-spring/dist";
-import { branchFromCommit } from "@atomist/sdm/api-helper/goal/executeBuild";
-import { DelimitedWriteProgressLogDecorator } from "@atomist/sdm/api-helper/log/DelimitedWriteProgressLogDecorator";
-import { spawnAndWatch } from "@atomist/sdm/api-helper/misc/spawned";
+import {
+    CommonJavaGeneratorConfig,
+    HasSpringBootApplicationClass,
+    MavenBuilder,
+} from "@atomist/sdm-pack-spring/dist";
+import {executableJarDeployer} from "@atomist/sdm-pack-spring/dist/support/java/deploy/executableJarDeployer";
+import {ListLocalDeploys} from "@atomist/sdm-pack-spring/dist/support/maven/deploy/listLocalDeploys";
+import {SpringBootSuccessPatterns} from "@atomist/sdm-pack-spring/dist/support/spring/deploy/localSpringBootDeployers";
```

Comment:
Missing spaces. Did you run `npm run lint:fix` locally?

Reply:
`lint:fix` does not fix the spaces. It's an IDE setting.